docs/user-guide/catalog.md

```diff
@@ -150,6 +150,11 @@ In **General Settings** the user can set the title he wants to assign to this se
 
 In **Advanced Settings** the user can set, other than the standard options, also:
 
+* *Localized styles* (only for the WMS service) through which it is possible to change the language of the layers. In this way the layer's legend and the layer's labels on the map will appear localized according to the current language
```

Comment:
* *Localized styles* (only for the WMS service) if enabled allows to include the MapStore's locale in each **GetMap**, **GetLegendGraphic** and **GetFeatureInfo** requests to the server so that the WMS server, if properly configured, can use that locale to:
   - Use localized lables for Tiles in case of vector layers (the layer's style must be properly configured for this using the [ENV variable support](https://docs.geoserver.org/stable/en/user/styling/sld/extensions/substitution.html))
   - Produce a localized layer legend in case of vector layers (the layer's style must be properly configured to use the [Localized tag for rule titles](https://docs.geoserver.org/stable/en/user/styling/sld/language.html))
   - Produce a localized output for GetFeatureInfo requests (the freemarker template need to be properly configured to retrieve [the locale from the request](https://docs.geoserver.org/stable/en/user/tutorials/freemarker.html))

Enabling that option, all layers added to the map from this catalog source will be localized as described above (it is possible to tune again that setting for each single layer by opening the Layer Settings in TOC).

Reply:
Please put a link here to the other section
>...Layer Settings in TOC).